src/lib/select/select.spec.ts

```diff
@@ -449,6 +450,22 @@ describe('MdSelect', () => {
       expect(fixture.componentInstance.select.selected).not.toBeDefined();
     });
 
+    it('should be able to programmatically select a falsy option', () => {
+      fixture.destroy();
+
+      let falsyFixture = TestBed.createComponent(FalsyValueSelect);
+
+      falsyFixture.detectChanges();
+      falsyFixture.debugElement.query(By.css('.mat-select-trigger')).nativeElement.click();
```

Comment:
It's there since we're also doing an extra check that the selected option has the proper class.